src/pages/PersonaCreation/Page/Page.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import _ from 'lodash';
+import { Editor as EditorType } from 'slate/dist';
+
+// Import the Slate editor factory.
+import { createEditor } from 'slate';
+
+// Import the Slate components and React plugin.
+import { Slate, Editable, withReact, ReactEditor } from 'slate-react';
+import { PageWrapper } from 'components/PageWrapper';
+import styled from 'styled-components';
+import { TopNav } from 'components/TopNav/TopNav';
+import { BlockTools } from './BlockTools/BlockTools';
+import { Element } from './BlockTools/EditorFunctionalities/EditorFunctionalities';
+import { EditorContextProvider, EditorContextType } from './EditorContext';
+
+const StyledPageWrapper = styled(PageWrapper)`
+  position: relative;
+  padding: 0px;
+`;
+
+type ActiveToolsType = 'bloc' | 'inline' | false;
```

Comment:
why You are using boolean here?

Reply:
Because this type can have 3 values possibility.